### PR TITLE
Translated deploy-app-from-appstore.md

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Translated deploy-app-from-appstore.md.
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-user-guide/application/deploy-app-from-appstore.md [hunk 1]
```diff
@@ -1,86 +1,86 @@
...
 keywords: 'Kubernetes, chart, helm, KubeSphere, application, App Store'
-description: 'How to deploy apps from the App Store.'
-linkTitle: "Deploy Apps from App Store"
+description: '如何从应用商店部署应用。'
+linkTitle: "从应用商店部署应用"
 weight: 10130
 ---
 
-The [App Store](../../../application-store/) is also the public app repository on the platform, which means every tenant on the platform can view the applications in the Store regardless of which workspace they belong to. The App Store contains 15 featured enterprise-ready containerized apps and apps released by tenants from different workspaces on the platform. Any authenticated users can deploy applications from the Store. This is different from private app repositories which are only accessible to tenants in the workspace where private app repositories are imported.
+[应用商店](../../../application-store/)是平台上的公共应用仓库。平台上的每个租户，无论属于哪个企业空间，都可以查看应用商店中的应用。应用商店包含15个精选的企业就绪的容器化应用，以及平台上不同企业空间的租户发布的应用。任何经过身份验证的用户都可以从应用商店部署应用。这与私有应用仓库不同，访问私有应用仓库的租户必须属于私有应用仓库所在的企业空间。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/application/deploy-app-from-appstore.md:9]: 包含15个精选
->
包含 15 个精选

File: content/zh/docs/project-user-guide/application/deploy-app-from-appstore.md [hunk 2]
```diff
@@ -1,86 +1,86 @@
...
-This tutorial demonstrates how to quickly deploy [NGINX](https://www.nginx.com/) from the KubeSphere App Store powered by [OpenPitrix](https://github.com/openpitrix/openpitrix) and access its service through NodePort.
+本教程演示如何从基于 [OpenPitrix](https://github.com/openpitrix/openpitrix) 的 KubeSphere 应用商店快速部署 [NGINX](https://www.nginx.com/)，并通过 NodePort 访问其服务。
 
-## Prerequisites
+## 准备工作
 
-- You have enabled [OpenPitirx (App Store)](../../../pluggable-components/app-store).
-- You need to create a workspace, a project, and a user account (`project-regular`) for this tutorial. The account needs to be a platform regular user invited to the project with the `operator` role. For more information, see [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project/).
+- 您需要启用 [OpenPitirx (App Store)](../../../pluggable-components/app-store)。
+- 您需要创建一个企业空间、一个项目和一个用户帐户（例如 `project-regular`）。该帐户必须是已邀请至项目的平台普通用户，并具有 `operator` 角色。有关更多信息，请参见[创建企业空间、项目、帐户和角色](../../../quick-start/create-workspace-and-project/)。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/application/deploy-app-from-appstore.md:16]: 已邀请至项目的平台普通用户
->
已邀请至该项目的平台普通用户

**Review by Sherlock113 — COMMENTED** [review]
Please fix these two minor issues. Others look good to me. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**